src/data.js

```diff
@@ -19413,6 +19413,27 @@ module.exports = [
       'Heroku',
     ],
   },
+  {
+    name: 'Rasul Kireev',
+    description: 'Software Engineer at Readwise specialising in Django',
+    url: 'https://rasulkireev.com/uses',
+    country: '🇷🇺',
+    emoji: '🐍',
+    computer: 'macos',
```

Comment:
Hey @rasulkireev thanks for contributing, will you make one minor change:
"computer" must be one of [apple, windows, linux, bsd]

Reply:
Thanks, updated.